buildSrc/build.gradle.kts

```
@@ -4,8 +4,19 @@ plugins {
 
 repositories {
     mavenCentral()
+    google()
+}
+
+gradlePlugin {
+    plugins {
+        register("bugsnag-ndk-table-plugin") {
```

Is this a more descriptive name?

```suggestion
        register("bugsnag-jni-link-table-plugin") {
```

---

Yes, I think that's a much nicer name.